app/mailers/commontator/subscriptions_mailer.rb

```diff
@@ -12,39 +12,48 @@ def comment_created(comment, recipients)
 
   def setup_variables(comment, recipients)
     @comment = comment
-    comment_user = User.find(comment.creator_id).full_name
-    @comment_header = "#{comment_user} has commented on #{@comment.thread.commontable.name}"
-    @comment_header = "#{comment_user} has replied to a comment on #{@comment.thread.commontable.name}" unless @comment.parent_id.nil?
-    @thread = @comment.thread
-    @creator = @comment.creator
-    @mail_params = { from: @thread.config.email_from_proc.call(@thread) }
-    practice = Practice.find(@comment.thread.commontable_id)
-    support_network_email = practice.support_network_email || nil
-    @recipient_emails = recipients.map do |recipient|
-      support_network_email.present? && recipient === support_network_email.downcase ? recipient : Commontator.commontator_email(recipient, self)
-    end
-    @using_mailgun = Rails.application.config.action_mailer.delivery_method == :mailgun
```

Comment:
This param seems like it's still being used in `#setup_mail_params`?

Reply:
good catch! I forgot to follow up on the removal of this